_ci/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+echo "=========================== Starting Test&Deploy Script ==========================="
+PS4="\[\e[35m\]+ \[\e[m\]"
+set -vex
+pushd "$(dirname "${BASH_SOURCE[0]}")/../"
+
+# Always build the image, but only publish from the "master" branch
+[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ] && PROFILE="internal" || PROFILE="local"
+
+# If the branch is "master" and the commit is not a Pull Request then deploy the JAR SNAPSHOT artifacts
+[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ] && DEPLOY="deploy" || DEPLOY="test"
+
+# If the branch is "master" and the profile provided is not "aio-test" do not deploy alfresco-transformer-base
+[ "${1}" != "aio-test" ] && [ "${TRAVIS_BRANCH}" = "master" ] && SKIP_DUP_DEPLOY="-Dparent.deploy.skip=true -Dtransformer.base.deploy.skip=true" || SKIP_DUP_DEPLOY=""
```

Comment:
Why was this needed ?

Reply:
This was needed in order to prevent deploying multiple alfresco-transform-core and alfresco-transformer-base snapshots. I've discussed this with @eknizat and we feel we have come up with a better solution, separating the deployment of core and base to a separate job.